doc/lfsc1_5.g4

```diff
@@ -117,14 +117,14 @@ term
   | rat_const
   | '(' term term+ ')'
   | '(' '\\' iden term ')'
-  | '(' '@' iden term ')'
+  | '(' '@' iden term term ')'
   | '(' ':' type term ')'
   | '(' '%' iden type term ')'
 // Extensions ///////////////////
   | '(' 'lam' iden term ')'
   | '(' 'let' iden term term ')'
   | '(' 'proved-by' type term ')'
-  | '(' 'assuming' vtype+ term ')'
+  | '(' 'assuming' '(' (ntype | vtype)+ ')' term ')'
```

Comment:
Adding parentheses is a good idea here, just like for the `->` form.
It's even more important in this case, because most of the proof will likely be in the final term here, so we *really* don't want to construct the AST for it.